pages/content/amp-dev/documentation/courses/beginning-course/our-first-amp-page@pt_BR.md

```diff
@@ -0,0 +1,451 @@
+
+As seguintes tags são necessárias nas páginas AMP. As páginas AMP válidas devem:
+
+- Começar com o doctype `<! Doctype html>`.
+
+- Conter as tags `<head>` e `<body>`.
+
+- Conter uma tag `<meta charset =" utf-8 ">` como o primeiro filho da tag `<head>`.
+
+- Conter uma tag `<meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">` dentro de sua tag `<head>`. **Nota**: Também é recomendado incluir `initial-scale=1`.
+
+As regras a seguir são especificamente para configurar o AMP runtime. As páginas AMP válidas também devem:
+
+- Conter uma tag `<html ⚡>` de nível superior. O símbolo de raio indica que este é um site AMP. **Nota**: `<html amp>` também é aceito.
+
+- Conter a tag`<script async src="https://cdn.ampproject.org/v0.js"></script>` dentro da tag `<head>`. Isso carrega a biblioteca JavaScript AMP. **Nota**: Como prática recomendada, você deve incluir o script no`<head>`.
+
+- Conter uma tag `<link rel="canonical" href="$SOME_URL">` dentro de seu `<head>`.  Isso aponta para a versão HTML normal do seu site, se houver, ou aponta para si mesmo, se não houver uma versão não AMP do site. **Nota**: Você deve substituir `$SOME_URL` no atributo` href` acima pelo URL real de sua página.
+
+- Conter o código padrão de estilo AMP na tag `<head>`. Este CSS oculta o conteúdo da página até que a biblioteca AMP termine de carregar. O boilerplate de estilo AMP é o seguinte snippet:
+
+[sourcecode:html]
+{% raw %}
+<style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+{% endraw %}[/sourcecode]
+
+[tip type="note"]
+**Nota**: O boilerplate do AMP deve ser copiado e colado como está. O código está reduzido acima, mas também funciona se você estiver usando um formatador automático em seu código, como o Prettier. Certifique-se de não alterar a ordem do texto dentro das tags acima ou alterar os valores. Se você quiser saber mais sobre o que essas tags no boilerplate de AMP significam e por que são importantes, leia sobre elas no [apêndice](apêndice.md).
```

Comment:
Travis picked up on this broken link! You have to use the English name, `appendix.md`.

Reply:
Done!